File: Prog 3 - Trabalho 1 (Java)/src/trabalho1/Coligacao.java
package trabalho1;

import java.util.LinkedList;

public class Coligacao {
	
	private String nome;
	private LinkedList<Partido> partidosDaColigacao = new LinkedList<Partido>();
	private int votosColigacao, vereadoresEleitos;

	public LinkedList<Partido> getPartidosDaColigacao() {
		return partidosDaColigacao;
	}

	public void setPartidoDaColigacao(LinkedList<Partido> partidosDaColigacao) {
		this.partidosDaColigacao = partidosDaColigacao;
	}
	
	public int getVereadoresEleitosCol(){
		return vereadoresEleitos;
	}
	
	public void setVereadoresEleitosCol (int vereadoresEleitosCol){
		this.vereadoresEleitos = vereadoresEleitosCol;
	}
	
	public int getVotosColigacao (){
		return votosColigacao;
	}
	
	public void setVotosColigacao(int votosColigacao){
		this.votosColigacao = votosColigacao;
	}

	public String getNome() {
		return nome;
	}

	public void setNome(String nome) {
		this.nome = nome;
	}
}

File: Prog 3 - Trabalho 2 (C++)/Coligacao.cpp

#include "Coligacao.h"
#include "Partido.h"
#include "Vereador.h"
#include <string>


using namespace std;

void Coligacao::setNome (string nomeColigacao){
    nome = nomeColigacao;
}

string Coligacao::getNome (){
    return nome;
}

void Coligacao::addNovoPartido (Partido* novoPartido){
    partidosDaColigacao.push_back(novoPartido);
}

void Coligacao::setVotosColigacao (int votosTotalColigacao){
    votosColigacao = votosTotalColigacao;
}

int Coligacao::getVotosColigacao (){
    return votosColigacao;
}

void Coligacao::setVereadoresEleitos (int vereadoresTotalEleitos){
    vereadoresEleitos = vereadoresTotalEleitos;
}

int Coligacao::getVereadoresEleitos (){
    return vereadoresEleitos;
}

Coligacao::Coligacao() {
}

Coligacao::Coligacao(const Coligacao& orig) {
}

Coligacao::~Coligacao() {
}

